pkg/v6/configmap/desired.go

```diff
@@ -25,7 +24,12 @@ type IngressController struct {
 }
 
 type IngressControllerController struct {
-	Replicas int `json:"replicas"`
+	Replicas int                                `json:"replicas"`
+	Service  IngressControllerControllerService `json:"service"`
+}
+
+type IngressControllerControllerService struct {
+	Enabled bool `json:"enabled"`
```

Comment:
A separate flag is needed so this can be set to false when installing the temp resources.

https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/ab6f1b9e8fc68af22e97293b6c0141615c64063c/helm/kubernetes-nginx-ingress-controller-chart/charts/tempresources/templates/configmap.yaml#L48